src/hooks/sceneGraphNodes/sceneGraphNodes.ts

```diff
@@ -0,0 +1,86 @@
+import { useMemo } from 'react';
```

Comment:
I think maybe this file should be called "hooks" if we want to match the other pattern with "types" and "util"?

Reply:
Hm, perhaps. However, it is also already located in the hooks folder, and the other files with hooks in this folder have a name that explains their usage. That's why I named it like this when I created it. 

But I guess now that this file is moved into a subfolder, it might make sense to name it hooks instead. Might need some discussion. What does @engbergandreas think about it?